erpnext/manufacturing/doctype/routing/routing.py

```diff
@@ -11,6 +11,18 @@
 class Routing(Document):
 	def validate(self):
 		self.set_routing_id()
+	
+	def on_update(self):
+		self.update_operation_time_in_linked_BOMs()
```

Comment:
contrary to what ankush suggested, this is very non standard naming for our codebase
```suggestion
		self.update_operation_time_in_linked_boms()
```